.eslintrc.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
         ],
         "@typescript-eslint/no-use-before-define": [
           "error",
-          { functions: false },
+          { functions: false, classes: false },
```

Comment:
This kind of change should be in a separate commit...?

Also, unlike functions, classes are actually not hoisted so this rule makes sense for classes and less so for functions.

It works for you in this diff because the whole file is initialized long before the function is called. It's literally the same as putting a `let` after the function where you use it, which I'm pretty sure is something we don't want to allow either.

Reply:
You are right.